apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyCustomization.md

```diff
@@ -0,0 +1,29 @@
+You can add some of your own customizations to the default theme.
+For example, you can utilize `defaultFontStyle` to override the default font style and
```

Comment:
Sounds very weird when saying _…utilize `defaultFontStyle` to override the default font style…_
How about this? And you can use it or not or just take some ideas 😄 

`For example, when loading the theme you can override the `defaultFontStyle` property when an override of every font variant (small, medium, large, etc.) is wanted or you can target specific ones through the `fonts` property. The example bellow shows both scenarios that can be either be used separately or combined.

Reply:
Working from that: 

``For example, you can set the `defaultFontStyle` property to modify every font variant (small, medium, large, etc.), or you can target specific variants through the `fonts` property. These can be used separately or combined, as shown in the example below.``